### PR TITLE
Add prometheus and grafana dashboards

### PR DESCRIPTION
**Pull request by ivange94**

### CODE REVIEW

**Review by dessaya — APPROVED** [review]

File: tools/monitoring/config.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "database": {
+    "directory": "waspdb"
+  },
+  "logger": {
+    "level": "debug",
```
Comment (jorgemmsilva) [inline tools/monitoring/config.json:6]: ~maybe let's set `info` as the default? debug is too spammy for normal operations, I think~

edit: on a second thought I can see why we want users to run with debug logging for now

Comment (jorgemmsilva) [reply tools/monitoring/config.json:6]: btw, why not use the default `config.json` that is on the root of the project?

**Review by jorgemmsilva — APPROVED** [review]
Nice! 🚀

File: tools/monitoring/config.json (hunk 1 above)
Comment (ivange94) [reply tools/monitoring/config.json:6]: The config in the root project doesn't have the metrics plugin turned on. I wasn't sure if it's something we'll always want to have turned on for the root config thats why I created this one.

Comment (jorgemmsilva) [reply tools/monitoring/config.json:6]: okay, makes sense 👍

**Pull request merged**